python/ray/_private/ray_constants.py

```diff
@@ -124,6 +124,10 @@ def env_bool(key, default):
 # Number of units 1 resource can be subdivided into.
 MIN_RESOURCE_GRANULARITY = 0.0001
 
+# Set this environment variable to populate the dashboard URL with
+# and external hosted Ray dashboard URL.
+EXTERNAL_RAY_DASHBOARD_URL = "EXTERNAL_RAY_DASHBOARD_URL"
```

Comment:
Ray environment variables need to start with `RAY_`. I suggest `RAY_OVERRIDE_DASHBOARD_URL` or `RAY_EXTERNAL_DASHBOARD_URL`.